package/scripts/params.py

```diff
@@ -73,15 +72,18 @@
 else:
     hdfs_namespace = map_cdap_site['hdfs.namespace']
 
+cdap_security_enabled = config['configurations']['cdap-site']['security.enabled']
+
 # Kerberos stuff
+kerberos_enabled = config['configurations']['cluster-env']['security_enabled']
```

Comment:
just to doublecheck, should this be ``cdap_security_enabled`` ?  Else, i don't see where ``cdap_security_enabled`` 2 lines above is used... but I could be wrong...